independent-publisher-connectors/BitlyIP/apiDefinition.swagger.json

```diff
@@ -0,0 +1,10627 @@
+          "type": "string"
+        }
+      },
+      "required": [
+        "is_active"
+      ]
+    },
+    "BitlinkOverrideHistory": {
+      "title": "BitlinkOverrideHistory",
+      "type": "object",
+      "properties": {
+        "total_count": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "bitlinks": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BitlinkHistory"
+          }
+        }
+      }
+    },
+    "SSOSettings": {
+      "title": "SSOSettings",
+      "type": "object",
+      "properties": {
+        "slo_endpoint": {
+          "type": "string"
+        },
+        "default_group_guid": {
+          "type": "string"
+        },
+        "idp_url": {
+          "type": "string"
+        },
+        "certificate": {
+          "type": "string"
+        },
+        "organization_guid": {
+          "type": "string"
+        },
+        "saml_endpoint": {
+          "type": "string"
+        },
+        "identity_provider": {
+          "type": "string"
+        },
+        "domains": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "issuer_url": {
+          "type": "string"
+        },
+        "url_slug": {
+          "type": "string"
+        }
+      }
+    },
+    "LaunchpadButtonRequest": {
+      "title": "LaunchpadButtonRequest",
+      "type": "object",
+      "properties": {
+        "bitlink": {
+          "example": "https://bit.ly/GzmMT6",
+          "type": "string"
+        },
+        "title": {
+          "example": "Short links, big results",
+          "type": "string"
+        },
+        "is_active": {
+          "type": "boolean"
+        },
+        "is_pinned": {
+          "type": "boolean"
+        },
+        "schedule_end": {
+          "type": "string"
+        },
+        "long_url": {
+          "example": "https://www.google.com",
+          "type": "string"
+        },
+        "domain": {
+          "example": "bit.ly",
+          "type": "string"
+        },
+        "schedule_start": {
+          "type": "string"
+        }
+      }
+    },
+    "CreateShare": {
+      "title": "CreateShare",
+      "type": "object",
+      "properties": {
+        "text": {
+          "type": "string"
+        },
+        "bitlink": {
+          "type": "string"
+        },
+        "account_type": {
+          "$ref": "#/definitions/SocialAccounts"
+        },
+        "account_id": {
+          "type": "string"
+        },
+        "group_guid": {
+          "type": "string"
+        }
+      }
+    },
+    "WebhookUpdate": {
+      "title": "WebhookUpdate",
+      "example": {
+        "name": "My Webhook",
+        "url": "https://bitly.com/webhooks",
+        "group_guid": "Ba1bc23dE4F",
+        "is_active": true,
+        "organization_guid": "Oa1bcd234eF",
+        "client_id": "fakeclientid123",
+        "client_secret": "itsasecreticanttellyou",
```

Comment:
@troystaylor Could you please remove example sections which contains client id and secret properties and its values, This is blocking the connector production deployment.

Reply:
This has been done. Thanks, Srikanth.